### PR TITLE
MNT Use GitHub Actions CI

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/gha-ci/issues/11<br /><br />Workflow run https://github.com/creative-commoners/recipe-core/actions?query=branch%3Apulls%2F4.10%2Fmodule-standards

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,9 @@
         "silverstripe/mimevalidator": "2.3.x-dev"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.5"
+        "phpunit/phpunit": "^9.5",
+        "silverstripe/versioned": "^1",
+        "mikey179/vfsstream": "^1.6.10"
```
Comment (GuySartorelli) [inline composer.json:18]: Are we doing vfsstream via composer.json or via ci? It's currently added explicitly in the ci... should we remove it from the ci and only include it in composer.json for the modules that need it?

File: composer.json [hunk 2]
```diff
@@ -13,7 +13,9 @@
         "silverstripe/mimevalidator": "2.3.x-dev"
     },
     "require-dev": {
```
Comment (GuySartorelli) [inline composer.json:15]: travis config also includes `silverstripe/admin` and `silverstripe-themes/simple` - do we need those here?

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (emteknetnz) [reply composer.json:18]: Perhaps, though we can do that later, there's already an issue for that https://github.com/silverstripe/gha-ci/issues/6

File: composer.json (hunk 2 above)
Comment (emteknetnz) [reply composer.json:15]: I don't think so?  We're not running behat here so I don't see why we would.

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**